scripts/controls.js

```diff
@@ -42,8 +42,8 @@ export default class Controls {
 
   get _removeEffectsButton() {
     return {
-      name: 'remove-convenient-effects',
-      title: 'Remove Convenient Effects',
+      name: game.i18n.localize("Controls.removeconvenienteffects"),
```

Comment:
This also doesn't need to be localized. It's just the identifier

Reply:
Noted! Will undo